go.mod-orig

```diff
@@ -0,0 +1,25 @@
+module github.com/HewlettPackard/dws
```

Comment:
Are you intentionally retaining this file?

Reply:
nope, thanks